src/intervals/precision.jl

```diff
@@ -21,10 +21,12 @@ function bigequiv(a::Interval{T}) where {T <: AbstractFloat}
         return atomic(Interval{BigFloat}, a)
     end
 end
+bigequiv(a::Interval{<:Integer}) = Interval(big(a.lo), big(a.hi))
```

Comment:
This is the change I'm least sure of. This seems safer than `bigequiv(float(a))` but of course it will be much slower.